src/main/java/nl/_42/password/validation/PasswordHolder.java

```diff
@@ -0,0 +1,7 @@
+package nl._42.password.validation;
+
+public interface PasswordHolder {
+    Long getId();
```

Comment:
Not all applications use a `Long` identifier. I would keep this out of the interface and let the implementation cast to a specific sub-class